app/grandchallenge/algorithms/forms.py

```diff
@@ -13,18 +19,109 @@
     AlgorithmPermissionRequest,
     Job,
 )
+from grandchallenge.components.models import InterfaceKind
 from grandchallenge.core.forms import (
     PermissionRequestUpdateForm,
     SaveFormInitMixin,
     WorkstationUserFilterMixin,
 )
 from grandchallenge.core.validators import ExtensionValidator
-from grandchallenge.core.widgets import MarkdownEditorWidget
+from grandchallenge.core.widgets import JSONEditorWidget, MarkdownEditorWidget
 from grandchallenge.groups.forms import UserGroupForm
 from grandchallenge.jqfileupload.widgets import uploader
 from grandchallenge.jqfileupload.widgets.uploader import UploadedAjaxFileList
+from grandchallenge.reader_studies.models import ANSWER_TYPE_SCHEMA
 from grandchallenge.subdomains.utils import reverse_lazy
 
+image_upload_text = (
+    "The total size of all files uploaded in a single session "
+    "cannot exceed 10 GB.<br>"
+    "The following file formats are supported: "
+    ".mha, .mhd, .raw, .zraw, .dcm, .nii, .nii.gz, "
+    ".tiff, .png, .jpeg and .jpg.<br>"
+    "The following file formats can be uploaded and will be converted to "
+    "tif: Aperio(.svs), Hamamatsu(.vms, .vmu, .ndpi), Leica(.scn), MIRAX"
+    "(.mrxs) and Ventana(.bif)."
+)
+
+file_upload_text = (
+    "The total size of all files uploaded in a single session "
+    "cannot exceed 10 GB.<br>"
+    "The following file formats are supported: "
+)
```

Comment:
Would be good if we can remove this duplication somehow as we're always editing this text, but low priority.